ethereum/forge-scripts/DeployDummyContract.s.sol

```diff
@@ -0,0 +1,20 @@
+
+
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+import {Implementation} from "../contracts/Implementation.sol";
+import {Setup} from "../contracts/Setup.sol";
+import "forge-std/Script.sol";
+
+contract DeployDummyContract is Script {
+    function run(uint256 num) public {
+        vm.startBroadcast();
+        for(uint256 i=0; i<num; i++) {
+            deploy();
+        }
+        vm.stopBroadcast();
+    }
+    function deploy() internal {
+        Setup setup = new Setup();
```

Comment:
This could be just a simple message call instead of a deploy.
In fact, if we're using this only in devnet, we should probably look at [cheatcodes](https://book.getfoundry.sh/cheatcodes/set-nonce) that let us set the current nonce instead of spamming transactions.

Reply:
I think the cheatcode solution is actually not possible - played around a little and am unable to get it to permanently modify the nonce value in the anvil devnet blockchain - I think the cheat-code is something that only works for a forge-test like environment (but not certain)

Reply:
A shame :pensive: